quil-py/Makefile.toml

```diff
@@ -0,0 +1,21 @@
+[env]
+RUST_BACKTRACE = 0
+
+[tasks.poetry-install]
+command = "poetry"
+args = ["install"]
+
+[tasks.install-quil]
+command = "poetry"
+args = ["run", "maturin", "develop"]
+
+[tasks.stubtest]
```

Comment:
Should these be documented? ie in Contributing.md

Reply:
Yep! Good call. I've documented it. I've also made it so that `stubtest-flow` is part of the default flow, so that one can just run `cargo make` and get the appropriate validation on both crates.